webrtc-stats.html

```diff
@@ -607,9 +607,13 @@ <h3>
           RTCP SR or RR block.
         </p>
         <p>
-          The lifetime of all RTP [= monitored object =]s starts when the <a>RTP stream</a> is first
-          used: When the first RTP packet is sent or received on the <a>SSRC</a> it represents, or when
-          the first RTCP packet is sent or received that refers to the <a>SSRC</a> of the <a>RTP stream</a>.
+          The lifetime of all RTP [= monitored object =]s starts when the <a>RTP
+          stream</a> is first configured and the required <a>SSRC</a> attribute
+          is known. For send streams, the SSRC is decided when setting the SDP
+          offer. For receive streams, the SSRC is typically known after
+          receiving the first packet, however if the SSRC is signalled via SDP
+          then the RTP stream is created when the remote SDP is set containing
+          the SSRC information.
```

Comment:
Let's abandon this PR and just add an "if packets > 0" check to the implementation? This avoids the dependency on whether or not the SSRC was signalled and it also avoids complexity about "what if the offer was rolled back"

Reply:
You create them when you configure them, you destroy them when you rollback?

Reply:
@jan-ivar What does Firefox do today in the case that SSRC is signalled in SDP? Does it wait for packets > 0 or does it do what Chrome does and expose the outbound-rtp as soon as the SSRC is known?